packages/sdks/mitosis.config.cjs

```diff
@@ -198,6 +198,337 @@ const filterActionAttrBindings = (json, item) => {
+  'i',
+  'kbd',
+  'mark',
+  'q',
+  'rp',
+  'rt',
+  'ruby',
+  's',
+  'samp',
+  'small',
+  'span',
+  'strong',
+  'sub',
+  'sup',
+  'time',
+  'u',
+  'var',
+  'wbr',
+  'area',
+  'audio',
+  'img',
+  'map',
+  'track',
+  'video',
+  'embed',
+  'iframe',
+  'object',
+  'param',
+  'picture',
+  'portal',
+  'source',
+  'svg',
+  'math',
+  'canvas',
+  'noscript',
+  'script',
+  'del',
+  'ins',
+  'caption',
+  'col',
+  'colgroup',
+  'table',
+  'tbody',
+  'td',
+  'tfoot',
+  'th',
+  'thead',
+  'tr',
+  'button',
+  'datalist',
+  'fieldset',
+  'form',
+  'input',
+  'label',
+  'legend',
+  'meter',
+  'optgroup',
+  'option',
+  'output',
+  'progress',
+  'select',
+  'textarea',
+  'details',
+  'dialog',
+  'summary',
+  'slot',
+  'template',
+  // tags below are SVG tags. See the below article for list of SVG tags
+  // https://developer.mozilla.org/en-US/docs/Web/SVG/Element
+  'animate',
+  'animateMotion',
+  'animateTransform',
+  'circle',
+  'clipPath',
+  'defs',
+  'desc',
+  'discard',
+  'ellipse',
+  'feBlend',
+  'feColorMatrix',
+  'feComponentTransfer',
+  'feComposite',
+  'feConvolveMatrix',
+  'feDiffuseLighting',
+  'feDisplacementMap',
+  'feDistantLight',
+  'feDropShadow',
+  'feFlood',
+  'feFuncA',
+  'feFuncB',
+  'feFuncG',
+  'feFuncR',
+  'feGaussianBlur',
+  'feImage',
+  'feMerge',
+  'feMergeNode',
+  'feMorphology',
+  'feOffset',
+  'fePointLight',
+  'feSpecularLighting',
+  'feSpotLight',
+  'feTile',
+  'feTurbulence',
+  'filter',
+  'foreignObject',
+  'g',
+  'hatch',
+  'hatchpath',
+  'image',
+  'line',
+  'linearGradient',
+  'marker',
+  'mask',
+  'metadata',
+  'mpath',
+  'path',
+  'pattern',
+  'polygon',
+  'polyline',
+  'radialGradient',
+  'rect',
+  'set',
+  'stop',
+  'switch',
+  'symbol',
+  'text',
+  'textPath',
+  'tspan',
+  'use',
+  'view',
+];
+
+const ANGULAR_PLUGIN6 = () => ({
+  json: {
+    pre: (json) => {
+      if (VALID_HTML_TAGS.includes(json.name.toLowerCase())) {
+        json.name = `Builder${json.name}`;
+      }
+    },
+  },
+});
+
+const ANGULAR_PLUGIN7 = () => ({
+  json: {
+    pre: (json) => {
+      if (json.name === 'Columns' || json.name === 'Symbol') {
+        json.imports.push({
+          path: '../../components/dynamic-div',
+          imports: {
+            DynamicDiv: 'default',
+          },
+        });
+      }
+      if (json.name === 'ContentComponent' || json.name === 'Block') {
+        json.imports.push({
+          path: '../dynamic-div',
+          imports: {
+            DynamicDiv: 'default',
+          },
+        });
+      }
+      if (json.name === 'EnableEditor') {
+        json.imports.push({
+          path: '../../dynamic-div',
+          imports: {
+            DynamicDiv: 'default',
+          },
+        });
+      }
+    },
+  },
+});
+
+const ANGULAR_PLUGIN8 = () => ({
+  code: {
+    post: (code) => {
+      code = code.replace('"div"', 'DynamicDiv');
+      return code;
+    },
+  },
+});
+
```

Comment:
Why don't we add an `angular` key to the `useTarget`s, so that we don't need any of these workaround plugins?

For example in `Columns`:

```diff
+ import DynamicDiv from '../../dynamic-div.lite.tsx`;


         <DynamicRenderer
            key={index}
            TagName={
              column.link
                ? props.builderLinkComponent ||
                  useTarget({
                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                    // @ts-ignore
                    reactNative: BaseText,
+                  angular: DynamicDiv,
                    default: 'a',
                  })
                : useTarget({
                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                    // @ts-ignore
                    reactNative: View,
+                  angular: DynamicDiv,
                    default: 'div',
                  })
            }
```

Reply:
yes! this will be best, let me update it

Reply:
thanks!!